packages/utilities/src/enabledFeatures.ts

```diff
@@ -0,0 +1,21 @@
+import { IFabricConfig } from './IFabricConfig';
+import { getWindow } from './dom/getWindow';
+
+/**
+ * Used to determine if a portion of code should run.
+ * See https://github.com/microsoft/fluentui/wiki/enabledFeatures-Switches for more details.
+ * @param name - globally unique name associated with this switch
+ * @param owner - alias or name of the owner of this switch.
+ * @param date - date this switch was created, preferably MM/DD/YY e.g. 12/31/20, or longform e.g. Dec 31, 2020
+ */
+export function isFeatureEnabled(name: string, owner: string, date: string) {
```

Comment:
owner and date seems like there is no point.

Reply:
Why this this function needed at all?

We should consider a `getSettings().features.myFlag`, available at the suite.

Reply:
Let's do this:

1. Move or redefine the config in `oufr`. 
2. If there isn't one already add a `getSettings` helper to access FabricConfig (so renaming it later can be managed in one place)
3. The interface returned is type safe.

Then adding any features that need to be manually turned on can by typesafe.

Hope that makes sense. let me know if you have any qs or need any help.

Reply:
`owner` and `date` are there so we know who to track down about a feature that's been in the code, so we can have policies like "must graduate features in 6 months". It helps with maintenance.

Reply:
I think it would be cleaner to leave off `owner` and `date` here and add a wrapper in SP code which takes those.